requirements_dev.txt

```diff
@@ -6,4 +6,5 @@ coverage==7.2.7
 types-jsonschema==4.17.0.8
 types-requests==2.31.0.1
 types-boto3==1.0.2
+deepdiff==6.3.0
 jupytext==1.13.6
```

Comment:
@InnocentBug is `jupytext` for our documentation?

Reply:
yes, and no.
We use jupytext to convert the example `md` documents into python documents and run them.

To ensure, that the examples always work.